### PR TITLE
Added load config file functionality, fix issue #4588

### PR DESCRIPTION
**Pull request by AviadP**
loading new config file after OCP upgrade
fix issue #4588
Signed-off-by: aviadp <apolak@redhat.com>

### CODE REVIEW

**Review by ebenahar — DISMISSED** [review]

File: tests/ecosystem/upgrade/test_upgrade_ocp.py [hunk 1]
```diff
@@ -26,6 +31,34 @@ class TestUpgradeOCP(ManageTest):
...
+    def load_ocp_version_config_file(self, ocp_upgrade_version):
+        """
+        Loads config file to the ocs-ci config with upgrade version
+
+        Args:
+            ocp_upgrade_version: (str): version to be upgraded
+
+        """
+
+        short_ocp_upgrade_version = ".".join(ocp_upgrade_version.split(".")[0:2])
```
Comment (petr-balogh) [inline tests/ecosystem/upgrade/test_upgrade_ocp.py:43]: What about 4.10 4.12 and so on.

Will this work here?

Comment (petr-balogh) [inline tests/ecosystem/upgrade/test_upgrade_ocp.py:43]: What about 4.10 4.12 and so on.

Will this work here?

Comment (AviadP) [reply tests/ecosystem/upgrade/test_upgrade_ocp.py]: fixed

Comment (AviadP) [reply tests/ecosystem/upgrade/test_upgrade_ocp.py]: fixed

File: tests/ecosystem/upgrade/test_upgrade_ocp.py [hunk 2]
```diff
@@ -26,6 +32,35 @@ class TestUpgradeOCP(ManageTest):
     5. monitor cluster health
     """
 
+    def load_ocp_version_config_file(self, ocp_upgrade_version):
+        """
+        Loads config file to the ocs-ci config with upgrade version
+
+        Args:
+            ocp_upgrade_version: (str): version to be upgraded
```
Comment (petr-balogh) [inline tests/ecosystem/upgrade/test_upgrade_ocp.py:40]:
```suggestion
            ocp_upgrade_version (str): version to be upgraded
```

Comment (AviadP) [reply tests/ecosystem/upgrade/test_upgrade_ocp.py]: fixed

**Review by petr-balogh — APPROVED** [review]

**Review by OdedViner — APPROVED** [review]
LGTM

**Pull request merged**